src/css/flags/touch.less

```diff
@@ -21,5 +21,8 @@
     .jw-display-icon-container {
         pointer-events: none;
     }
-}
 
+    &.jwplayer&.jw-state-paused .jw-display-icon-container {
```

Comment:
because we want to select .jwplayer.jwflag-touch.jw-state-paused.
ampersands refers to parent selectors.